internal_testsuites/golang/testsuite/stream_logs_test/stream_logs_test.go

```diff
@@ -1,8 +1,3 @@
-//go:build !kubernetes
```

Comment:
can we also add a test in the circle ci yaml to docker / k8s basic CLI tests

Reply:
yep

Reply:
seems like k8s basic cli tests are turned off right now due to inconsistencies like this onehttps://github.com/kurtosis-tech/kurtosis/issues/1866. I'm going to merge this to get the fix out ASAP and make a note to resolve the CI dicrepancy